packages/eslint-plugin-pf-codemods/src/rules/v6/emptyStateHeaderMoveIntoEmptyState/emptyStateHeader-move-into-emptyState.ts

```diff
@@ -0,0 +1,267 @@
+      }
+
+      if (value.type === "JSXExpressionContainer") {
+        return context.getSourceCode().getText(value.expression);
+      }
+
+      return "";
+    };
+
+    const getNodesText = (nodes: Node[]) => {
+      return nodes
+        .map((node) => context.getSourceCode().getText(node))
+        .join("");
+    };
+
+    const getElementChildText = (children: JSXElement["children"]) => {
+      if (!children.length) {
+        return "";
+      }
+
+      switch (children[0].type) {
+        case "JSXText":
+          return (children as JSXText[]).map((child) => child.value).join("");
+        case "JSXExpressionContainer":
+        case "JSXSpreadChild":
+          return getNodesText(
+            children.map(
+              (child) =>
+                (child as JSXExpressionContainer | JSXSpreadChild).expression
+            )
+          );
+        case "JSXElement":
+        case "JSXFragment":
+          return getNodesText(children as JSXElement[] | JSXFragment[]);
+        default:
+          return "";
+      }
+    };
+
+    return {
+      JSXElement(node: JSXElement) {
+        if (!isComponentNode(node, "EmptyState")) {
+          return;
+        }
+
+        const header = getChildElementByName("EmptyStateHeader", node);
+        const emptyStateTitleTextAttribute = getAttribute(node, "titleText");
+
+        if (!header && !emptyStateTitleTextAttribute) {
+          context.report({
+            node,
+            message: `${baseMessage} You must manually supply a titleText prop to EmptyState.`,
+          });
+          return;
+        }
+
+        if (!header || header.type !== "JSXElement") {
+          return;
+        }
+
+        const headingClassNameAttribute = getAttribute(header, "className");
+        const headingLevelAttribute = getAttribute(header, "headingLevel");
+        const titleClassNameAttribute = getAttribute(header, "titleClassName");
+        const titleTextAttribute = getAttribute(header, "titleText");
+        const headerIconAttribute = getAttribute(header, "icon");
+
+        const headerChildren = header.children;
+
+        if (!titleTextAttribute && !headerChildren.length) {
+          context.report({
+            node,
+            message: `${baseMessage} You must manually supply a titleText prop to EmptyState, then you can rerun this codemod.`,
+          });
+          return;
+        }
+
+        if (titleTextAttribute && headerChildren.length) {
+          context.report({
+            node,
+            message: `${baseMessage} Because the children for EmptyStateHeader are now inaccessible you must remove either the children or the titleText prop, then you can rerun this codemod.`,
+          });
+          return;
+        }
+
+        const headingClassNameValue = getAttributeValueText(
+          headingClassNameAttribute
+        );
+
+        const headingClassName = headingClassNameValue
+          ? `headerClassName="${headingClassNameValue}"`
```

Comment:
This should be modified to address the suggested change of `getAttributeValueText` functionality.
```suggestion
          ? `headerClassName=${headingClassNameValue}`
```